packages/api/tests/endpoints/test-granules.js

```diff
@@ -2116,240 +2094,21 @@ test.serial('PATCH replaces an existing granule in all data stores', async (t) =
   );
 });
 
-test.serial('PUT does not update non-current-timestamp undefined fields for existing granules in all datastores', async (t) => {
```

Comment:
Should we update the deleted `PUT` tests to `PATCH` tests?

Reply:
Looking at this, it looks like a possible merge issue 😦

Reply:
re-added these, updated to the correct endpoints and fixed a few other things in the units: 

59239b9fc3
ed3f2ea666